package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "@powsybl/diagram-viewer",
+  "version": "0.0.1",
+  "description": "Typescript library to integrate a powsybl svg diagram in a javascript project.",
+  "author": "powsybl team",
+  "homepage": "https://github.com/powsybl",
```

Comment:
link

Reply:
not sure maybe https://www.powsybl.org/ ?

Reply:
Yes, and github repo is below in the file